commit 8b782cb7514aa3a09503e848e4c9ac3662a37075 (tag: v1.8.0, refs/changes/02/302/2)
Author: David Pacheco <dap@joyent.com>
Date:   2016-08-16T21:32:01+00:00 (3 years, 2 months ago)
    
    joyent/node-verror#42 release 1.8.0
    Reviewed by: Alex Wilson <alex.wilson@joyent.com>
    Approved by: David Pacheco <dap@joyent.com>

diff --git a/CHANGES.md b/CHANGES.md
index c2a1c4f..3ee585e 100644
--- a/CHANGES.md
+++ b/CHANGES.md
@@ -2,6 +2,10 @@
 
 ## Not yet released
 
+No changes yet.
+
+## v1.8.0
+
 * #23 Preserve original stack trace(s)
 
 ## v1.7.0
diff --git a/package.json b/package.json
index 9ee1c8a..c370b32 100644
--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
 	"name": "verror",
-	"version": "1.7.0",
+	"version": "1.8.0",
 	"description": "richer JavaScript errors",
 	"main": "./lib/verror.js",
 	"repository": {
